nislmigrate/migrators/systems_management_migrator.py

```diff
@@ -11,7 +11,7 @@
 
 PKI_DIRECTORY_NAME = 'pki'
 PILLAR_DIRECTORY_NAME = 'pillar'
-SALT_PATH = os.path.join(get_ni_application_data_directory_path(), 'National Instruments', 'salt')
```

Comment:
This needed to be removed after submitting the sys mgmt migrator PR but never got pushed